app/src/main/java/com/example/mona/facebookoffline/EditActivity.java

```diff
@@ -41,15 +57,100 @@ public void onClick(View v) {
             public void onClick(View v) {
```

Comment:
These `View.OnClickListener`s are getting too big to put in the `onCreate` methods. I'm not sure the best way to mitigate this, but I think it would be nicer if we made a functional methods that returned the `postButtonClickListener` and the `saveButtonClickListener` etc. How does that sound to you?